dbt_project.yml

```diff
@@ -0,0 +1,36 @@
+name: 'snowplow_media_player'
+version: '0.1.0'
+config-version: 2
+require-dbt-version: ">=1.0.0"
+
+profile: 'default'
+
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+docs-paths: ["docs"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+  - "target"
+  - "dbt_packages"
+
+vars:
+  snowplow__percent_progress_boundaries: [10, 25, 50, 75]
+  snowplow__valid_play_sec: 30
+  snowplow__complete_play_rate: 0.99
+  snowplow__max_media_pv_window: 2
```

Comment:
Should you add a default entry for `snowplow__incremental_materialization`?

Reply:
I'd rather keep the variables at their intended package level for more clarity within scoping but good point, I have added a comment to the overview docs to leave the default materialisation intact when configuring the web package variables.